recipes/azure-servicemanagement-legacy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.20.1" %}
+
+package:
+  name: azure-servicemanagement-legacy
+  version: {{ version }}
+
+source:
+  fn: azure-servicemanagement-legacy-{{ version }}.zip
+  url: https://pypi.python.org/packages/5b/4b/013c5b8e782a7b744a6d92a5eb33e884615b9101bace5ddfd73c2e16fef3/azure-servicemanagement-legacy-{{ version }}.zip
```

Comment:
Please change to `https://pypi.io/packages/source/a/azure-servicemanagement-legacy/azure-servicemanagement-legacy-{{ version }}.zip`